cerberus-web/src/main/java/com/nike/cerberus/event/listener/AuditLoggingEventListener.java

```diff
@@ -24,8 +24,8 @@
 
 /** Event listener implementation that logs each event. */
 @Slf4j
+@ConditionalOnProperty("cerberus.events.auditLoggingEventListener.enabled")
```

Comment:
This should be in the config class that then optionally scans everything in `com.nike.cerberus.audit.athena`
The config class should be in `com.nike.cerberus.config`

Reply:
- [ ] move @ConditionalOnProperty to AthenaAuditLoggerConfiguration